src/main/java/com/redhat/devtools/intellij/quarkus/run/QuarkusRunConfiguration.java

```diff
@@ -121,24 +122,36 @@ private void allocateLocalPort() {
     @Nullable
     @Override
     public RunProfileState getState(@NotNull Executor executor, @NotNull ExecutionEnvironment environment) throws ExecutionException {
-        TelemetryMessageBuilder.ActionMessage telemetry = TelemetryService.instance().action(TelemetryService.RUN_PREFIX + "run");
-        telemetry.property("kind", executor.getId());
-        BuildToolDelegate toolDelegate = BuildToolDelegate.getDelegate(getModule());
+        Module module = getModule();
+
+        TelemetryMessageBuilder.ActionMessage telemetryMessage = TelemetryManager.instance()
+                        .action(TelemetryEventName.RUN_RUN);
+
+        if (telemetryMessage != null) {
+            telemetryMessage.property("kind", executor.getId());
+        }
```

Comment:
just create a simple Map to collect the event properties

Reply:
fixed